src/tasks/deploy-utils.js

```diff
@@ -66,9 +66,18 @@ function validateId(settings) {
  * @memberof slate-cli.tasks.watch, slate-cli.tasks.deploy
  * @private
  */
-
 gulp.task('validate:id', () => {
-  const file = fs.readFileSync(config.tkConfig, 'utf8');
+  let file;
+
+  try {
+    file = fs.readFileSync(config.tkConfig, 'utf8');
+  } catch (err) {
```

Comment:
You might want to handle this err a little bit. You could just have a permission error on your config file and then we're telling the user that it doesn't exist.

I think it would make sense to say if err is `ENOENT` then do this,
 otherwise, actually throw the error so that the user could debug the problem.
 (https://nodejs.org/api/errors.html#errors_common_system_errors)